superset/assets/src/dashboard/stylesheets/components/column.less

```diff
@@ -28,7 +28,6 @@
 
 .dashboard--editing {
   .grid-column:after {
-    border: 1px solid transparent;
```

Comment:
lol, what was this?

Reply:
There are a bunch of fairly convoluted styles around the dashed borders in the dashboard editor, which seem like they could use a simplification effort (a future PR, probably). I'll double check that this one wasn't providing some useful 1px offset.